CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 Changes to Joyride
 
 ## [Unreleased]
+- Fix: [Not disposing of disposables on dev reload](https://github.com/BetterThanTomorrow/joyride/issues/87)
```

Comment:
```suggestion

- Fix: [Not disposing of disposables on dev reload](https://github.com/BetterThanTomorrow/joyride/issues/87)
```

I think the release script might be sensitive to the blank line missing. (I should know. since I wrote the script, but anyway.)